java/ql/src/semmle/code/java/security/HardcodedCredentials.qll

```diff
@@ -37,10 +40,8 @@ class HardcodedExpr extends Expr {
   }
 }
 
-/**
- * An argument to a sensitive call, expected to contain credentials.
- */
-abstract class CredentialsSink extends Expr {
+/** An argument to a sensitive call, expected to contain credentials. */
+abstract class CredentialsSink extends DataFlow::Expr {
```

Comment:
```suggestion
abstract class CredentialsSink extends Expr {
```
The type `Expr` is not intentionally exposed through the `DataFlow` module (though with all the `import java` statements it's no surprise that it slipped in).